tap_hubspot/hubspot.py

```diff
@@ -6,6 +6,7 @@
 from datetime import datetime, timezone
 from typing import Dict, Iterable, Optional, DefaultDict, Set, List, Any, Tuple
 from dateutil import parser
+import simplejson
```

Comment:
I feel like `json` is a more standard library to catch this error. Is there a reason why you want to use `simplejson` instead?

```
from json.decoder import JSONDecodeError
```

Reply:
From `Response.json()` method, see that it raises a `ValueError`. 
```python
    def json(self, **kwargs):
        r"""Returns the json-encoded content of a response, if any.

        :param \*\*kwargs: Optional arguments that ``json.loads`` takes.
        :raises ValueError: If the response body does not contain valid json.
        """
```

I would not use `ValueError` since it might catch unwanted cases. Using either `simplejson` or `json` modules will do the work. However, I feel like `json` is a better known library than `simplejson`.

Reply:
But if I try to catch a json.decoder.JSONDecodeError it will not catch the simplejson.scanner.JSONDecodeError right?

Reply:
So the json method only raises `ValueError`. That is why both `json.decoder.JSONDecodeError` and `simplejson.scanner.JSONDecodeError` can catch the exception.

Reply:
yup the ValueError

Reply:
Both ways work. Just notice that `simplejson` is not as used as `json` library.